### DISCUSSION TITLE
old version of Termux from google store

### DISCUSSION THREAD

**Discussion by Gozer-404** [termux/termux-app#2317] [Q&A] [answered]
Hi,

maybe I didn't really understand the stuff:
- my termux version always display the extensive message saying that the version is too old and that I should check https://github.com/termux/termux-app#installation
- https://github.com/termux/termux-app#installation states that apk can be fetched from github: "Termux application can be obtained on Github either from Github Releases for >= **v0.118**" 
- on github (https://github.com/termux/termux-app/releases) I can see that the latest version is **v0.117 (F-Droid only)**

Finally where is the apk for v0.118 ?

**Comment by agnostic-apollo** [comment, marked as answer]
The currently latest version is still `v0.117`. Download it from F-Droid https://f-droid.org/en/packages/com.termux

If you want Github sources, you can download pre-release versions from `Github Build` action workflows as mentioned.
